.jenkins/rocm/build_and_test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# exit immediately on failure, or if an undefined variable is used
+set -eux
+
+FBGEMM_REPO_DIR=${1:-/workspace/FBGEMM}
+
+git config --global --add safe.directory $FBGEMM_REPO_DIR
+git config --global --add safe.directory $FBGEMM_REPO_DIR/third_party/asmjit
+git config --global --add safe.directory $FBGEMM_REPO_DIR/third_party/cpuinfo
+git config --global --add safe.directory $FBGEMM_REPO_DIR/third_party/googletest
+git config --global --add safe.directory $FBGEMM_REPO_DIR/third_party/hipify_torch
```

Comment:
There are some internal linter issues. Would you mind fixing them?

**[(info) Double quote to prevent globbing and word splitting]**
Above five lines, can I ask you to add double quotes? For example,`"$FBGEMM_REPO_DIR/..."` instead of `$FBGEMM_REPO_DIR`.

Reply:
The double quotes were added.